microcosm_flask/conventions/health.py

```diff
@@ -56,8 +63,9 @@ def configure_health(graph):
     """
 
     health = Health(name=graph.metadata.name)
+    path = graph.config.health.path_prefix + singleton_path_for(Health)
 
-    @graph.flask.route("/api/health")
+    @graph.route(path, Operation.Retrieve, Health)
```

Comment:
APIs that use the operation convention can use a slightly different route decorator that specifies the path, operation, and resource/class and does the rest according to conventions.